### ISSUE TITLE
AnalyticalTable: Cell not reading column name at the end

### ISSUE THREAD

**Issue by Arora05** [SAP/ui5-webcomponents-react#3189]
**AnalyticalTable: Cell not reading column name at the end**
The column name is only read out when the user moves horizontally i.e. one column to another. But when the user moves vertically i.e. same column up and down only the cell name is being read out not the cell name with the column header

**Expected behavior**
Ideally, the column header name should be read out every time user moves to another cell irrespective of movement

**Screenshots**
![Screenshot (318)](https://user-images.githubusercontent.com/89779778/177134664-e2b09fc0-298b-41d9-8a52-2a1cc041bced.png)
